CHANGELOG.md

```diff
@@ -45,6 +45,7 @@ This section is for maintaining a changelog for all breaking changes for the cli
 ### Fixed
 - Fixed error when deserializing a normalizer without 'type' ([#1111](https://github.com/opensearch-project/opensearch-java/pull/1111))
 - Fixed deserialization of SearchRequest when `_source` is an array ([#1117](https://github.com/opensearch-project/opensearch-java/pull/1117))
+- Removed null constraints on NodeInfo fields that are returned null from Amazon OpenSearch Service ([#1131](https://github.com/opensearch-project/opensearch-java/pull/1131))
```

Comment:
I think we can simplify this and follow the verbiage of the particular section ("Fixed")
```suggestion
- Fixed handling of NodeInfo fields that are not returned by Amazon OpenSearch Service ([#1131](https://github.com/opensearch-project/opensearch-java/pull/1131))
```